ci_framework/roles/cifmw_cephadm/tasks/pools.yml

```diff
@@ -23,24 +23,28 @@
     - name: Get ceph_cli
       ansible.builtin.include_tasks: ceph_cli.yml
 
-    - name: Create pool
+    - name: Create RBD pools
       ansible.builtin.command:
         cmd: >-
           {{ cifmw_cephadm_ceph_cli }}
           osd pool create
           {{ item.name }} {{ item.pg_num | default('') }}
           {{ item.rule_name | default('replicated_rule') }}
           --autoscale-mode {{ item.pg_autoscale | default('on') }}
+      when:
```

Comment:
care to add the condition after the `name:` key?

Reply:
done